### PR TITLE
test: small improvements when testing environment variables in Managers

### PR DESCRIPTION
**Pull request by kyle-cochran**

### CODE REVIEW

File: test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.test.cpp [hunk 1]
```diff
@@ -55,6 +91,14 @@ class OpenSpaceToolkit_Physics_Coordinate_Frame_Providers_IERS_Manager : public
...
 
     Manager& manager_ = Manager::Get();
+
+    const char* localRepositoryVarName = "OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY";
+    const char* fullDataVarName = "OSTK_PHYSICS_DATA_LOCAL_REPOSITORY";
+    const char* modeVarName = "OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_MODE";
+
+    char* localRepositoryPath;
+    char* fullDataPath;
+    char* modeValue;
```
Comment (vishwa2710) [inline test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.test.cpp:101]:
```suggestion
    const char* localRepositoryVarName_ = "OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_LOCAL_REPOSITORY";
    const char* fullDataVarName_ = "OSTK_PHYSICS_DATA_LOCAL_REPOSITORY";
    const char* modeVarName_ = "OSTK_PHYSICS_COORDINATE_FRAME_PROVIDERS_IERS_MANAGER_MODE";

    char* localRepositoryPath);
    char* fullDataPath_;
    char* modeValue_;
```
Nit: should we keep these as underscores as private/protected?

**Review by vishwa2710 — APPROVED** [review]
Ultra nit

File: test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.test.cpp [hunk 2]
```diff
@@ -41,6 +41,42 @@ class OpenSpaceToolkit_Physics_Coordinate_Frame_Providers_IERS_Manager : public
...
+        fullDataPath = std::getenv(fullDataVarName);
+        modeValue = std::getenv(modeVarName);
+    }
+
+    virtual void TearDown()
+    {
+        // reset cached environment variables
+        if (fullDataPath)
+        {
+            setenv(fullDataVarName, fullDataPath, true);
```
Comment (apaletta3) [inline test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.test.cpp:56]: `std::setenv` no? and same for `unsetenv`?

**Review by apaletta3 — COMMENTED** [review]
Not really sure all that is going on here, but it looks reasonable to me lol.

File: test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.test.cpp (hunk 2 above)
Comment (kyle-cochran) [reply test/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.test.cpp:56]: It's not in the Cpp standard, so likely not:
https://stackoverflow.com/questions/30292642/c-standard-library-stdsetenv-vs-setenv

**Review by apaletta3 — APPROVED** [review]
LGTM

**Pull request merged**